mobile/src/main/java/com/wedriveu/mobile/app/MainActivity.java

```diff
@@ -0,0 +1,124 @@
+package com.wedriveu.mobile.app;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.View;
+import com.wedriveu.mobile.R;
+import com.wedriveu.mobile.login.router.LoginRouter;
+import com.wedriveu.mobile.login.view.LoginView;
+import com.wedriveu.mobile.login.view.LoginViewImpl;
+import com.wedriveu.mobile.login.viewmodel.LoginViewModel;
+import com.wedriveu.mobile.login.viewmodel.LoginViewModelImpl;
+import com.wedriveu.mobile.model.Vehicle;
+import com.wedriveu.mobile.tripscheduling.router.SchedulingRouter;
+import com.wedriveu.mobile.tripscheduling.view.SchedulingView;
+import com.wedriveu.mobile.tripscheduling.view.SchedulingViewImpl;
+import com.wedriveu.mobile.tripscheduling.viewmodel.SchedulingViewModel;
+import com.wedriveu.mobile.tripscheduling.viewmodel.SchedulingViewModelImpl;
+import com.wedriveu.mobile.util.Constants;
+import com.wedriveu.mobile.util.location.LocationService;
+import com.wedriveu.mobile.util.location.LocationServiceImpl;
+
+/**
+ *
+ * @author Marco Balsassarri, Nicola Lasagni
```

Comment:
is this javadoc syntax corrent? shouldn't we use 
@author Nicola Lasagni
@author Marco Baldassarri
in order to generate a correct javadoc file?